gnocchi/tests/test_incoming.py

```diff
@@ -0,0 +1,65 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright © 2017 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import threading
+import uuid
+
+import numpy
+
+from gnocchi import incoming
+from gnocchi import storage
+from gnocchi.tests import base as tests_base
+
+
+class TestIncomingDriver(tests_base.TestCase):
```

Comment:
TestIncomingNotifierDriver? something less generic... same with filename.

Reply:
The incoming tests we have right now are in mixed with storage in `test_storage` for historical reason. This is the first test that is incoming specific. That's why it's called `test_incoming`.

My point: is this file and class should already exist and I'd be just adding `test_iter_on_sacks_to_process` to that. But it does not (for historical reason) so I'm just fixing it here. :)